demos/drawer/persistent-drawer.html

```diff
@@ -95,6 +95,9 @@
       <header class="mdc-toolbar mdc-elevation--z4">
         <div class="mdc-toolbar__row">
           <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
+            <span class="catalog-back">
```

Comment:
Nit: .catalog-back has a padding-right which I think is a bit too much here, given there's another toolbar button. Can you remove the padding here?

Reply:
Yes, I agree. Will remove